pygac/tests/test_angles.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2014-2019 Pytroll Developers
+
+# Author(s):
+
+#   Nina Hakansson <nina.hakansson@smhi.se>
+#   Adam Dybbroe <adam.dypbroe@smhi.se>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test function for the angle calculation.
```

Comment:
One line docstrings should have the closing """ on the same line. (yeah, I know pygac has a lot to be improved with styling, one more reason to start now :) )